File: Team2/Sristi Sharma/26.10.20/missingnumber.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int total = (nums.size()) * (nums.size() + 1) / 2;
        int sum=0;
    for (int i = 0; i < nums.size(); i++)
        sum+=nums[i];
    return total-sum;
        
    }
};

File: Team1/Riya Singh/missingNumber.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0;
        for(int i=0; i<nums.length; i++){
            sum = sum + nums[i];
        }
        int total = (nums.length )*(nums.length + 1)/2;
        return (total - sum);
    }  
}
